src/lib/autocomplete/autocomplete-foundation.ts

```diff
@@ -302,7 +302,7 @@ export class AutocompleteFoundation extends ListDropdownAwareFoundation implemen
       case 'ArrowDown':
         evt.preventDefault();
         if (!this._isDropdownOpen) {
-          this._showDropdown({ activateFirst: true });
+          this._showDropdown({ activateFirst: true, activateSelected: true });
```

Comment:
Yeah this was the best alternative I could think of without altering the flow since we can't query the dropdown or even the selected option state consistently at this point in time... These parameters operate more like "hey dropdown I'd like to request that this happens when you're ready" instead of forcing it.

Reply:
I'm okay with it, just wanted to mention it.